TenantFile/Api/TenantFile.Api.csproj

```diff
@@ -8,6 +8,25 @@
     <UserSecretsId>2e183d23-5e31-4ab7-bd0d-d8ecef7a6760</UserSecretsId>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Remove="Migrations\20200828160717_ToTableTenant.cs" />
```

Comment:
I think your IDE probably added these. We want to keep the csproj clean, so let's remove them

Reply:
Yeah this was something VS 2019 added. issue here https://github.com/dotnet/efcore/issues/18381 Should be safe to remove.